main.tf

```
@@ -40,6 +40,7 @@ locals {
     db_name          = local.db_info.database_name
     ce_pkg           = var.ce_pkg
     ee_pkg           = var.ee_pkg
+    ee_creds         = var.ee_creds_ssm_param
```

Might be better to include the reference to SSM params - otherwise might be mistaken for the actual values rather than SSM parameters

---

I'm not sure I follow. That's defined at the point of declaring the variable.

---

Apologies - would of been a bit clearer with an example;

```suggestion
    ee_creds_ssm_param         = var.ee_creds_ssm_param
```

It's basically to avoid confusion between unexpanded and expanded parameters, as you need to pull the values for the creds from SSM. Whereas for example `ee_pkg` contains the actual value